cosmosdb/common/ps-account-failover-priority-update.ps1

```diff
@@ -1,14 +1,39 @@
-# Change the failover priority for an Azure Cosmos Account
-# Assume West US = 0 and East US = 1, the script below will flip them
-# Updating location with failoverPriority = 0 will trigger a failover
+# Change the failover priority for a single-master Azure Cosmos Account
+# Note: Updating location with failoverPriority = 0 triggers a failover to the new region
+
+$location = "West US 2"
 $resourceGroupName = "myResourceGroup"
-$accountName = "mycosmosaccount"
+$accountName = "mycosmosaccount" # must be lower case.
```

Comment:
This token is not unique. You have checked the following __required__ box: 

> All identifiers which must be universally unique are guaranteed to be so.

Please only check boxes if you have actually completed those tests.

The best way to ensure a random identifier in PowerShell is to use the `Get-Random` cmdlet.

Reply:
@sptramer do you mean the existing "mycosmosaccount" name should be appended with the random number generated with the Get-Random cmdlet? 

This script was tested in  PowerShell 5.1 (Windows), I see it's unchecked now. Is that because we need to retest with the above change or is there something else missing?

Reply:
Yes, that's correct. The pattern that I've seen other submitters following is to use something like "cosmosaccount-$(Get-Random)" when creating a new resource.

I should be clear that using random identifiers for `Get-`only operations isn't necessary, since the intent of the script makes it implicit that users are responsible for already knowing the name of their existing resource.